include/avif/avif.h

```diff
@@ -1603,6 +1603,14 @@ typedef struct avifEncoder
     // Version 1.2.0 ends here. Add any new members after this line.
     // --------------------------------------------------------------------------------------------
 
+    // creationTime and modificationTime are only used when encoding an image sequence (animated
+    // image).
+    //
+    // creationTime and modificationTime are in seconds since midnight, Jan. 1, 1970 UTC (the Unix
+    // epoch). If set to 0 (the default), libavif uses the current time.
```

Comment:
It seems that only `modificationTime` should default to the current time. A better default for `creationTime` seems to be the modification time.

This way libavif clients that want deterministic encoder output will only need to set `modificationTime` in most cases.

What do you think?